e2e/publications.spec.ts

```diff
@@ -132,14 +132,14 @@ test.describe("Given a Video Publication link", async () => {
 test.describe("Given a Publication link with `by` attribution param", async () => {
   test.describe("When opening it", async () => {
     test("Then it should show the specified app first", async ({ videoPost }) => {
-      await videoPost.openAsSharedBy("lenstube");
+      await videoPost.openAsSharedBy("Tape");
 
       await expect(videoPost.options).toHaveText([
-        "Lenstube",
         "Buttrfly",
         "Collectz",
         "Hey",
         "Soclly",
+        "Tape",
```

Comment:
Oh, Tube should be first in this case (see `videoPost.openAsSharedBy("Tape")`).

I mentioned this in another PR, the `by` param is currently case sensitive... the correct link for now should have the `by=tape` as the `appId`. Can you please amend line 135 and move `Tape` to first position in this test?

Said that i think Lens Share should allows this to be case insensitive.

Reply:
https://lens-share-git-rename-to-tape-lens-bean.vercel.app/p/0x28a2-0x05ca?by=tape

resolved 🙇🏻